samples/sql-bridge/sample.md

```diff
@@ -9,6 +9,8 @@ related:
 
 ## Sql Bridge (Transport Integration)
 
+WARNING: This sample has been deprecated. Please refer to the [MsmqToSqlRelay Sample](/samples/msmqtosqlrelay/.) 
```

Comment:
Because when I incorrectly used sample.md in the url, i received the following message:

Error:'Please link to the sample directory and not the sample.md. ie use '/samples/msmqtosqlrelay/'.'. Url:'/samples/msmqtosqlrelay/sample.md'.